tests/php/Emitter/OutputEmitter/NodeEmitter/ApplyEmitterTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PhelTest\Emitter\OutputEmitter\NodeEmitter;
+
+use Phel\Ast\ApplyNode;
+use Phel\Ast\DefNode;
+use Phel\Ast\LiteralNode;
+use Phel\Ast\PhpVarNode;
+use Phel\Ast\TupleNode;
+use Phel\Emitter\OutputEmitter;
+use Phel\Emitter\OutputEmitter\Munge;
+use Phel\Emitter\OutputEmitter\NodeEmitter\ApplyEmitter;
+use Phel\Emitter\OutputEmitter\NodeEmitterFactory;
+use Phel\Emitter\OutputEmitter\SourceMap\SourceMapGenerator;
+use Phel\Lang\Symbol;
+use Phel\Lang\Table;
+use Phel\NodeEnvironment;
+use PHPUnit\Framework\TestCase;
+
+final class ApplyEmitterTest extends TestCase
+{
+    private ApplyEmitter $applyEmitter;
+
+    public function setUp(): void
+    {
+        $this->applyEmitter = new ApplyEmitter(
+            new OutputEmitter(
+                $enableSourceMaps = true,
+                new SourceMapGenerator(),
+                new NodeEmitterFactory(),
+                new Munge()
+            )
+        );
+    }
+
+    public function testPhpVarNodeAndFnNodeIsInfix(): void
+    {
+        $node = new PhpVarNode(NodeEnvironment::empty(), '+');
+        $args = [
+            new TupleNode(NodeEnvironment::empty(), [
+                new LiteralNode(NodeEnvironment::empty(), 2),
+                new LiteralNode(NodeEnvironment::empty(), 3),
+                new LiteralNode(NodeEnvironment::empty(), 4),
+            ]),
+        ];
+
+        $arrayNode = new ApplyNode(NodeEnvironment::empty(), $node, $args);
+        $this->applyEmitter->emit($arrayNode);
+
+        $this->expectOutputString(
+            'array_reduce([...((\Phel\Lang\Tuple::createBracket(, , );) ?? [])], function($a, $b) { return ($a + $b); });'
+        );
+    }
+
+    public function testPhpVarNodeButNoInfix(): void
+    {
+        $node = new PhpVarNode(NodeEnvironment::empty(), 'str');
+        $args = [
+            new LiteralNode(NodeEnvironment::empty(), 'abc'),
+            new LiteralNode(NodeEnvironment::empty(), 'def'),
```

Comment:
The last argument of `apply` must always be a sequence (`Tuple`). Therefore this line must be replaced with this:
```
new TupleNode(NodeEnvironment::empty(), [
  new LiteralNode(NodeEnvironment::empty(), 'def')
])
``